### ISSUE TITLE
Initial Value with Spaces

### ISSUE THREAD

**Issue by cmmcguire** [sodenn/react-fluent-edit#124]
First of all, this is a really great package and exactly what I was looking for!

Question - 
I have the `MuiFluentEdit` component working with mentions. However, I need to provide an initialValue with a first name and last name. As this includes a space between the first and last name, only the first name is recognized as part of the mention

![image](https://user-images.githubusercontent.com/48422935/219401870-72d8c7f5-9dfa-4be4-ac0f-addf029c9f8d.png)

Is there a way to serialize/deserialize the value in a way to make this work such that both the first and last name are included in the mention?

I noticed the beforeSerialize and afterDeserialize options but have not figured out how to use these for this purpose. 

Thanks!

Below is the code for my component:
```
import { useState } from "react";

// ** MUI Imports
import { Box } from "@mui/material";

// ** React-Fluent-Edit Imports
import { FluentEditProvider, Plugin } from "@react-fluent-edit/core";
import {
  createMentionsPlugin,
  MentionCombobox,
  MentionsProvider,
} from "@react-fluent-edit/mentions";
import { MuiFluentEdit } from "@react-fluent-edit/mui";

type CustomFluentEditorProps = {
  items: { text: string; trigger: string }[];
  onChange: (value: string) => void;
  initialValue?: string;
};

const plugin = createMentionsPlugin({
  mentions: [
    {
      trigger: "@",
      style: {
        color: "#17501b",
        backgroundColor: "#c8efcb",
      },
    },
  ],
});

const CustomFluentEditor = (props: CustomFluentEditorProps) => {
  return (
    <FluentEditProvider providers={[<MentionsProvider />]}>
      <Editor {...props} />
    </FluentEditProvider>
  );
};

const Editor = ({
  items,
  onChange,
  initialValue,
}: CustomFluentEditorProps) => {
  return (
    <Box>
      <MuiFluentEdit
        placeholder="What do you want to say?"
        initialValue={initialValue || ""}
        onChange={onChange}
        plugins={[plugin]}
        multiline={true}
        autoCorrect="off"
        autoCapitalize="off"
        spellCheck={false}
      >
        <MentionCombobox items={items} />
      </MuiFluentEdit>
    </Box>
  );
};

export default CustomFluentEditor;

```

**Comment by cmmcguire** [comment]
Additionally, is there a way to disable adding mentions that aren't explicitly provided as `items` to the `MentionCombobox`?

For example, I would like to disable the 'Add "C"' option in the example screenshot below. Thanks!

![image](https://user-images.githubusercontent.com/48422935/219469361-ae3801fc-4058-48aa-b9b5-c6a14a4ff0d9.png)

**Comment by sodenn** [comment]
> Question - I have the `MuiFluentEdit` component working with mentions. However, I need to provide an initialValue with a first name and last name. As this includes a space between the first and last name, only the first name is recognized as part of the mention

Hi @cmmcguire and sorry for the slow reply. Unfortunately, this is a technical limitation. Mentions are identified with a regular expression based on spaces. We could make the expression overridable with a configuration parameter. But I think it's challenging as a new pattern is needed to identify the end of a mention.

> Additionally, is there a way to disable adding mentions that aren't explicitly provided as items to the MentionCombobox?

I can add a configuration to only allow mentions in `items`. That would change the following:
- The "Add..." menu item is not visible
- Mentions are not added via the blur event if they are not in `items`
- Mentions in `initialValue` are treated as normal text and are not highlighted if they are not in `items` (maybe difficult to implement because the `items` prop is not known when deserializing)

Is it the behavior you would expect?

**Comment by cmmcguire** [comment]
Thank you for getting back to me @sodenn !

> But I think it's challenging as a new pattern is needed to identify the end of a mention.

No worries on the first question. I can get around this by putting an underscore instead of a space between first and last name (or any number of similar options).

> Is it the behavior you would expect?

For the second question, yes, this is the behavior I would expect, and the addition of this option would be excellent. 

The first two bullet points ("The "Add..." menu item is not visible" and "Mentions are not added via the blur event if they are not in `items`") would be great additions (and necessary for this feature to work for my use case). The third bullet point ("Mentions in `initialValue `are treated as normal text and are not highlighted if they are not in `items`") is not explicitly necessary for my use case since I can control what is passed in as `initialValue`.

Thank you once again!

**Comment by sodenn** [comment]
I published a beta version: `3.0.2-feat-disable-creatable.0`
Storybook: https://react-fluent-edit-git-feat-disable-creatable-sodenn.vercel.app/?path=/story/fluentedit--mentions
Configuration:
```
const mentionsPlugin = createMentionsPlugin({
  mentions: [ ... ],
  disableCreatable: true
})
```

**Comment by cmmcguire** [comment]
@sodenn - Excellent - that is perfect! Thank you!
